### PR TITLE
It 371  accessibility-updates

### PR DESCRIPTION
**Pull request by MHealton**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
     <!--- Social Media Logos and Links. --->
     <div class="footer__social-media footer__block d-flex py-2">
       {% for app in site.data.socialMedia %}
-        <a href="{{ app.link }}" target="_blank" class="px-1">
```
Comment (bwuphan) [inline _includes/footer.html:29]: Small deal but I've been new lining on the attributes when there are 3 or more of them or they have over 80 column width. See line 47 of this same file for reference.

File: _includes/navigation.html [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {% if page.url == "/" %}
 <nav id="navigation-bar" class="navbar navbar-expand-lg navbar-dark py-1 position-absolute navbar-fixed-top fixed-top navbar-inverse bg-inverse navigation-bar-dark">
   <a href="/" class="p-0 d-flex align-items-center">
-    <img class="navigation-bar__bg-logo" src="/assets/images/BG/logo_03.png">
```
Comment (bwuphan) [inline _includes/navigation.html:4]: comma unnecessary

File: _includes/navigation.html [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+    <img class="navigation-bar__bg-logo" src="/assets/images/BG/logo_03.png", alt="BG Software Logo">
   </a>
 {% else %}
 <nav id="navigation-bar" class="navbar navbar-expand-lg navbar-light shadow-sm py-1 position-relative">
   <a href="/" class="p-0 d-flex align-items-center">
-    <img class="navigation-bar__bg-logo" src="/assets/images/BG/logo_02.png">
```
Comment (bwuphan) [inline _includes/navigation.html:9]: Comma unnecessary.

File: _includes/navigation.html [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+    <img class="navigation-bar__bg-logo" src="/assets/images/BG/logo_02.png", alt="BG Software Logo">
   </a>
 {% endif %}
-  <button class="navbar-toggler" type="button" data-toggle="collapse" >
```
Comment (bwuphan) [inline _includes/navigation.html:12]: Comma unnecessary.

File: _includes/navigation.html [hunk 5]
```diff
@@ -53,11 +53,13 @@
           href="{{ app.link }}"
           target="_blank"
           class="px-1 navigation-bar__link"
+          name="{{ app.name }}"
+          aria-label="{{ app.description }}"
         >
           <i class="navigation-bar__icon hover-color-{{ app.name }} fa {{ app.img-class }} font-size-2"></i>
         </a>
         {% endfor %}
-        <a href="mailto:customsoftware@benningfieldgroup.com" class="px-1 navigation-bar__link">
```
Comment (bwuphan) [inline _includes/navigation.html:60]: Newline the attributes.

File: _layouts/post.html [hunk 6]
```diff
@@ -6,7 +6,7 @@
 <div class="blog-post mt-6 d-flex flex-column justify-content-center font-size-md pb-6">
   <div class="blog-post__content mx-auto w-100">
     <div class="blog-post__image-container pb-5 text-center">
-      <img src="/assets/images/blog/{{ page.image }}" class="blog-post__image rounded-lg">
```
Comment (bwuphan) [inline _layouts/post.html:9]: Newline the attributes.

File: _posts/2019-05-31-launching-a-business.md [hunk 7]
```diff
@@ -3,6 +3,7 @@ layout      : post
 title       : It's Time for the World to Hear from Us
 description : How often does one get to take an existing company and treat it like a startup?
 image       : 2019-05-31-its-time-for-the-world-to-hear-from-us.jpg
+image-description: "Link to blog post. Photo of night sky with trail of light from fireworks."
```
Comment (bwuphan) [inline _posts/2019-05-31-launching-a-business.md]: Keep the spacing between keys and values consistent.

File: _posts/2019-06-04-emotional-software.md [hunk 8]
```diff
@@ -3,6 +3,7 @@ layout      : post
 title       : Why a Software Purchase Is Absolutely an Emotional One
 description : ...and doesn't making the right call feel awesome?
 image       : 2019-06-04-why-a-software-purchase-is-absolutely-an-emotional-one.jpg
+image-description: "Link to blog post. Smiling man with his face pointed up. His eyes are closed and his hands are clenched and held up to his chest."
```
Comment (bwuphan) [inline _posts/2019-06-04-emotional-software.md:6]: Keep the spacing between keys and values consistent.

File: _posts/2019-06-07-questions-for-your-software-builder.md [hunk 9]
```diff
@@ -4,6 +4,7 @@ title       : If Your Software Builder Can’t Answer These 3 Questions, Ghost T
 description : These three will determine if your software build is going to be exciting, or
               a headache.
 image       : 2019-06-07-if_your_software_builder_cant_answer_these_3_questions_ghost_them.jpg
+image-description: "Link to blog post. Photo of chalkboard with a large question mark drawn in the center."
```
Comment (bwuphan) [inline _posts/2019-06-07-questions-for-your-software-builder.md]: Keep the spacing between keys and values consistent.

File: _posts/2019-06-10-folsom-where-we-work-and-play.md [hunk 10]
```diff
@@ -4,6 +4,7 @@ title       : "Folsom: Where We Work (and Play)"
 description : The story of our hometown, a former Gold Rush village
               tuned-major-suburb, and why our quality of life is amazing.
 image       : 2019-06-10-folsom-where-we-work-and-play.jpg
+image-description: "Link to blog post. Photo of Folsom Lake from the shore in the daytime. Tree branches in foreground"
```
Comment (bwuphan) [inline _posts/2019-06-10-folsom-where-we-work-and-play.md]: Keep the spacing between keys and values consistent.

File: _posts/2019-06-13-software-security.md [hunk 11]
```diff
@@ -3,6 +3,7 @@ layout      : post
 title       : "Security and Your Software: Are You Safe?"
 description : How to find out if you’re not.
 image       : 2019-06-13-security-and-your-software-are-you-safe.jpg
+image-description: "Link to blog post. Cartoon image of man in bank robber clothing walking out of laptop screen, carrying a bag over his shoulder, and a small flashlight opinted at images of credit cards."
```
Comment (bwuphan) [inline _posts/2019-06-13-software-security.md]: Keep the spacing between keys and values consistent.

File: _posts/2019-06-18-google-calendar-outage.md [hunk 12]
```diff
@@ -3,6 +3,7 @@ layout      : post
 title       : "What We Learned When Google Went Down (Twice)"
 description : It can happen to anyone..
 image       : 2019-06-18-what-we-learned-when-google-went-down-twice.jpg
+image-description: "Link to blog post. Photo of Yellow diamond-shaped road sign. Text reads 'DEAD END'"
```
Comment (bwuphan) [inline _posts/2019-06-18-google-calendar-outage.md]: Keep the spacing between keys and values consistent.

File: _posts/2019-06-26-software-costs.md [hunk 13]
```diff
@@ -4,6 +4,7 @@ title       : What’s My Software Going to Cost?
 description : The correct answer is, "It depends." But it's not anything like buying
               a car!
 image       : 2019-06-26-whats-my-software-going-to-cost.jpg
+image-description: "Link to blog post. Photo of computer screen with html code going across it."
```
Comment (bwuphan) [inline _posts/2019-06-26-software-costs.md]: Keep the spacing between keys and values consistent.

File: _posts/2019-09-24-software-security-time-to-turn-off-the-light.md [hunk 14]
```diff
@@ -3,6 +3,7 @@ layout: post
 title: "Software Security: Time to Turn Off the Light "
 description: "Don't make it easy for a cyber thief. Here are some simple ways to protect your business."
 image: 2019-09-24-software-security-time-to-turn-off-the-light.jpg
+image-description: "Link to blog post. Photo of green neon sign with arrow and the word 'ENTER'"
```
Comment (bwuphan) [inline _posts/2019-09-24-software-security-time-to-turn-off-the-light.md]: Keep the spacing between keys and values consistent.

File: _includes/navigation.html [hunk 15]
```diff
@@ -1,15 +1,15 @@
...
   </a>
 {% else %}
 <nav id="navigation-bar" class="navbar navbar-expand-lg navbar-light shadow-sm py-1 position-relative">
   <a href="/" class="p-0 d-flex align-items-center">
-    <img class="navigation-bar__bg-logo" src="/assets/images/BG/logo_02.png">
+    <img class="navigation-bar__bg-logo" src="/assets/images/BG/logo_02.png" alt="BG Software Logo">
   </a>
 {% endif %}
-  <button class="navbar-toggler" type="button" data-toggle="collapse" >
+  <button class="navbar-toggler" aria-label="Open the menu" aria-hidden=true type="button" data-toggle="collapse" >
```
Comment (bwuphan) [inline _includes/navigation.html]: Should newline these.

File: _layouts/post.html [hunk 16]
```diff
@@ -6,7 +6,7 @@
...
+      <img src="/assets/images/blog/{{ page.image }}" class="blog-post__image rounded-lg" alt="{{ page.title }}">
```
Comment (bwuphan) [inline _layouts/post.html]: Newline these attributes.

**Review by bwuphan — APPROVED** [review]

File: _posts/2019-06-26-software-costs.md [hunk 17]
```diff
@@ -1,12 +1,13 @@
...
-comments    : true
-tags        : [John Scott, Software]
+image             : 2019-06-26-whats-my-software-going-to-cost.jpg
+image-description : "Link to blog post. Photo of computer screen with HTML code going across it."
+author            : John Scott
+comments          : true
+tags              : [John Scott, Software]
 ---
 
 When you’re talking business with a customer, there are two basic questions which will always be asked: “What’s this going to cost?” and “”How does this save me time and money?” Business decisions are made using logic, facts and data/research. A solution either is going to work, or not.
```
Comment (aajungroth) [inline _posts/2019-06-26-software-costs.md]: Extra double quote before how does this save me time and money?

**Comment by MHealton** [comment]
Good catch. correction made

File: _posts/2019-06-10-folsom-where-we-work-and-play.md [hunk 18]
```diff
@@ -1,12 +1,13 @@
 ---
```
Comment (aajungroth) [inline _posts/2019-06-10-folsom-where-we-work-and-play.md:1]: This is still a good post, but it's misleading now that we've moved. Should we update the post with a note that we've moved?

**Review by aajungroth — APPROVED** [review]

**Pull request merged**